pkg/reconciler/taskrun/validate_resources_test.go

```diff
@@ -422,3 +423,118 @@ func TestValidateResolvedTaskResources_InvalidResources(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateTaskSpecRequestResources_ValidResources(t *testing.T) {
+	ctx := context.Background()
+
+	tcs := []struct {
+		name string
+		task *v1beta1.TaskSpec
+	}{{
+		name: "no-requested-resources",
```

Comment:
nit: These names can include spaces, they'll be converted to `_`s automatically if the `t.Run` fails. Spaces might make them easier to read in code.

Reply:
sure!